frameworks/hdfs/tools/zone-resolver.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+export DATA_NODE_ZONE=$(curl $SCHEDULER_API_HOSTNAME/v1/state/zone/data/$1)
```

Comment:
A comment or two in this file explaining what $1 is, the purpose of the script, etc would be lovely.